alire.toml

```diff
@@ -10,4 +10,4 @@ website = "https://github.com/Fabien-Chouteau/atomic"
 tags = ["atomic", "spark", "embedded", "nostd"]
 
 [configuration.variables]
-Backend = {type = "Enum",  values = ["Intrinsic", "armv6m"], default = "Intrinsic"}
+Backend = {type = "Enum",  values = ["Intrinsic", "armv6m", "rp2040"], default = "Intrinsic"}
```

Comment:
Maybe use the name `rp2040_spinlock` here. It will be more explicit about the technique used.

Reply:
That's a good idea. I've renamed it in commit f45d23a